### PR TITLE
systemd-machine-id-setup: add page

### PR DESCRIPTION
**Pull request by quantumflo**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/systemd-machine-id-setup.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# systemd-machine-id-setup
+
+> Initialize the machine ID stored in `/etc/machine-id` at install time, with a provisioned or randomly generated ID.
```
Comment (gutjuri) [inline pages/linux/systemd-machine-id-setup.md:3]:
```suggestion
> Initialize the machine ID stored in `/etc/machine-id` at install time with a provisioned or randomly generated ID.
```

File: pages/linux/systemd-machine-id-setup.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+> Note: Always use sudo to execute these commands as they require elevated privileges.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-machine-id-setup.html>.
+
+- Print the machine ID generated or committed:
```
Comment (gutjuri) [inline pages/linux/systemd-machine-id-setup.md:7]:
```suggestion
- Print the generated or committed machine ID:
```

File: pages/linux/systemd-machine-id-setup.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+`systemd-machine-id-setup --print`
+
+- Specify an image policy:
+
+`systemd-machine-id-setup --image-policy={{your_policy}}`
+
+- Display output as JSON:
```
Comment (gutjuri) [inline pages/linux/systemd-machine-id-setup.md:15]:
```suggestion
- Display the output as JSON:
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I've added some suggestions

**Review by gutjuri — APPROVED** [review]

File: pages/linux/systemd-machine-id-setup.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# systemd-machine-id-setup
+
+> Initialize the machine ID stored in `/etc/machine-id` at install time with a provisioned or randomly generated ID.
+> Note: Always use sudo to execute these commands as they require elevated privileges.
```
Comment (kbdharun) [inline pages/linux/systemd-machine-id-setup.md:4]:
```suggestion
> Note: Always use `sudo` to execute these commands as they require elevated privileges.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

----

Btw @quantumflo, when creating new PRs I would suggest basing the branches head to `tldr-pages:main` instead of selecting your fork. That way you won't have unnecessary commits in your PR (and in the final commit message).

**Pull request merged**